### ISSUE TITLE
release on pypi

### ISSUE THREAD

**Issue by gmelodie** [wh00hw/pyFlipper#7]
Hey there! Love your project. It would be great if we could `pip install pyflipper`. Are you planning on realeasing the project on [pypi](https://pypi.org/)?

**Comment by wh00hw** [comment]
Hi, thanks for suggesting. It was planned but delayed due to lack of time. I've just published the package, now you can install with `pip install pyflipper`

**Comment by gmelodie** [comment]
Oh amazing!! Thank you so much!

**Comment by Vaasref** [comment]
Hi, it seems that the version on PyPI is out of date.

Here is a how the pyflipper.date.date() function looks like :
```
def date(self) -> datetime:
    result = self._serial_wrapper.send("date")
    #FIXME: Flipper returns isoweekday 1-7 but %w is 0-6
    result = f"{result[:-1]}{int(result[-1])-1}"
    return datetime.strptime(result, "%Y-%m-%d %H:%M:%S %w")
```

**Comment by wh00hw** [comment]
hey @Vaasref thanks for noticing, even if that's not version problem but an old bug. I've just fixed and published a new version (0.18). Be sure to upgrade with `pip install pyflipper --upgrade`
